pglast/printers/ddl.py

```diff
@@ -2291,12 +2522,14 @@ def rename_stmt(node, output):
     elif objtype == OT.OBJECT_ATTRIBUTE:
         output.write('ATTRIBUTE ')
         output.print_name(node.subname)
+    elif objtype == OT.OBJECT_DOMCONSTRAINT:
+        output.writes('CONSTRAINT')
+        output.print_name(node.subname)
     output.swrite('TO ')
     output.print_name(node.newname)
     if node.behavior == enums.DropBehavior.DROP_CASCADE:
         output.write(' CASCADE')
 
-
```

Comment:
Please use two empty lines to separate toplevel functions, for consistency and easier navigation.